pkg/controllers/provisioning/v1alpha1/reallocation/annotate.go

```diff
@@ -71,6 +71,7 @@ func (a *Annotator) CordonNodes(ctx context.Context, nodes []*v1.Node) error {
 	for _, node := range nodes {
 		persisted := node.DeepCopy()
 		node.Spec.Unschedulable = true
+		node.Labels[v1alpha1.ProvisionerDrainingKey] = "draining"
```

Comment:
Convention is to use `string(true)`

Reply:
I wanted to have the draining key be able to filtered on either `draining` or `drained`